newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js

```diff
@@ -39,19 +39,40 @@ const resourceKindToInputAcceptedMimes = {
   tilemap: ['application/json'],
   tileset: ['application/json'],
   bitmapFont: [],
-  model3D: ['model/gltf-binary'],
+  model3D: [
+    'file',
+    // The following mime type is not handled by Safari. The verification will be handled
+    // after the files have been picked.
+    // 'model/gltf-binary'
+  ],
 };
 
-export const getInputAcceptedMimesAndExtensions = (
-  resourceKind: ResourceKind
-) => {
+const getFileNameExtension = (fileName: string): string => {
+  return fileName.split('.').reverse()[0];
```

Comment:
Consider using `path.extname` from `path-browserify` (NOT from `optionalRequire('path')`). Note that `path.extname("something.glb")` will return `.glb`